### PR TITLE
ACMS-4081: Fixing schema for editors as per CKeditor5.

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMS-4081

**Proposed changes**
Fixing schemas for editors as per CKEditor5.

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good.

File: modules/acquia_cms_common/src/Facade/ConfigHandlerFacade.php [hunk 1]
```diff
@@ -120,4 +120,25 @@ public function processThirdPartySettings(array $entity, array $settings): void
...
+   *
+   * @return void
+   *   Saving the configuration.
+   */
+  public function updateConfigByKey(string $config, string $key, array $value): void {
+    $config = $this->configFactory->getEditable($config);
+    if ($config->get($key)) {
+      $config->set($key, $value);
+    }
+    $config->save();
```
Comment (chandan-singh7929) [inline modules/acquia_cms_common/src/Facade/ConfigHandlerFacade.php:141]: Why this `$config->save();` is out side of if statement?

Comment (chandan-singh7929) [reply modules/acquia_cms_common/src/Facade/ConfigHandlerFacade.php:141]: Can't we use existing function `_acquia_cms_common_update_page_configurations()` for this purpose?

**Review by chandan-singh7929 — COMMENTED** [review]
Overall changes looks good to me, though I have provided minor suggestion, please take a look.

File: modules/acquia_cms_common/src/Facade/ConfigHandlerFacade.php (hunk 1 above)
Comment (apathak18) [reply modules/acquia_cms_common/src/Facade/ConfigHandlerFacade.php]: Idea is to use configHandlerFacade class to update the config related data.

**Review by deepakmishra2 — APPROVED** [review]

**Review by chandan-singh7929 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Comment by vishalkhode1** [comment]
All CI should pass post #1858 is merged. Blocking merge for this unless, until #1858 is merged in `develop` branch.

**Pull request merged**